### PR TITLE
ConversionContext: less spammy logs

### PR DESCRIPTION
**Pull request by dhalperi**
Don't log for nothing to report.

### CODE REVIEW

File: projects/batfish/src/main/java/org/batfish/main/Batfish.java [hunk 1]
```diff
@@ -2151,16 +2152,16 @@ private void serializeConversionContext(
         cpServerData = readAllInputObjects(keys, snapshot);
       }
       if (!cpServerData.isEmpty()) {
+        LOGGER.info("\n*** READING CHECKPOINT MANAGEMENT CONFIGS ***\n");
```
Comment (dhalperi) [inline projects/batfish/src/main/java/org/batfish/main/Batfish.java:2155]: eh not thrilled about this change actually, removes I/O from log.

**Review by SLarkworthy — COMMENTED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/batfish/batfish/8995#-:-NwCS-S17N_sYi5ih018:bnfp4nl)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @dhalperi)

___
*[`projects/batfish/src/main/java/org/batfish/main/Batfish.java` line 2155 at r1](https://reviewable.io/reviews/batfish/batfish/8995#gh-1577038364:-NwCRua48ntJmlRDeqbT:b-uj078b) ([raw file](https://github.com/batfish/batfish/blob/0ec4d76de2837a86cc94062a0fbba841282b01c1/projects/batfish/src/main/java/org/batfish/main/Batfish.java#L2155)):*
<details><summary><i>Previously, dhalperi (Dan Halperin) wrote…</i></summary><blockquote>

eh not thrilled about this change actually, removes I/O from log.
</blockquote></details>

Could add a separate info log if this is useful?

File: projects/batfish/src/main/java/org/batfish/main/Batfish.java (hunk 1 above)
Comment (dhalperi) [reply projects/batfish/src/main/java/org/batfish/main/Batfish.java:2154]: Mitigated by separating listing keys and reading data. The listing is fast, so it's "okay" that's hidden in logs IMO.

**Review by dhalperi — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/batfish/batfish/8995)* status: all files reviewed, 1 unresolved discussion (waiting on @SLarkworthy)

**Review by SLarkworthy — COMMENTED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/batfish/batfish/8995#-:-NwLb86Q7vkZVFmLPGMq:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @dhalperi)

**Pull request merged**